### ISSUE TITLE
`potion_master_cleric.sc` can sometimes give the same trade in the same villager, and gives too few xp

### ISSUE THREAD

**Issue by altrisi** [gnembon/scarpet#325]
By testing the script I've found it gave me the same trade (Potion of Grace) twice in the same villager and stage, with different prices.
A good enhancement could be for it to not be able to dupe the trade, given you'll obviously go for the cheaper one anyway.

I also think the potion trades give too little xp, according to the level and that they're not the simplest trades (by requiring an extra item).

**Comment by rv3r** [comment]
What do you mean by "dupe the trade"? Two of any, say, haste potions or precisely two haste potions, both of the splash variety? Either way, very doable. Xp change is also easy enough.

**Comment by altrisi** [comment]
Dupe the trade I meant they could get the trade twice in the list iirc, just for different prices.

**Comment by rv3r** [comment]
Got it. How does changing it to not get the same combo (potion effect and type) ever sound?

**Comment by altrisi** [comment]
Makes sense, though if it's random idk if it may cause them to have very different prices, unless you overlap the prices (note: can't remember if it was random).

**Comment by rv3r** [comment]
I don't see an option in villager trade data to mess with the amount of xp given, so I think the best I could do is hook into the event and summon more xp orbs.
